exec/java-exec/src/main/resources/rest/options.ftl

```diff
@@ -21,6 +21,26 @@
 <#macro page_head>
     <script type="text/javascript" language="javascript"  src="/static/js/jquery.dataTables-1.10.16.min.js"> </script>
     <script type="text/javascript" language="javascript" src="/static/js/dataTables.colVis-1.1.0.min.js"></script>
+    <script>
+        //Alter System Values
+        function alterSysOption(optionName, optionValue, optionKind) {
+            $.post("/option/"+optionName, {kind: optionKind, name: optionName, value: optionValue}, function () {
+                location.reload(true);
+            });
+        }
+
+        //Read Values and apply
+        function alterSysOptionUsingId(optionRawName) {
```

Comment:
Looks like before we just had POST method on the Update button and did not have to do replacements. I would prefer if e stick to previous approach or define one without parsing and replacements.

Reply:
It does make the changes a little more complex, but the primary motivation was that the same code path is being exercised for making any changes to the options. It's also why I kept it as 2 commits. We can revert it if we don't want a common execution.

Reply:
Looks like common execution brings more complexity than should, please revert.

Reply:
Ok. Will do that and run the tests.